build_runner/lib/src/package_graph/apply_builders.dart

```diff
@@ -17,7 +17,7 @@ import 'package_graph.dart';
 import 'target_graph.dart';
 
 typedef BuildPhase BuildPhaseFactory(String package, BuilderOptions options,
-    InputSet targetSources, InputSet generateFor);
+    InputSet targetSources, InputSet generateFor, bool isReleaseBuild);
```

Comment:
I think this is technically breaking.  `BuildPhaseFactory` isn't public itself, but there is a public field on `BuilderApplication` which exposes it...